app/backend/templates/v2/host_reference.mjml

```diff
@@ -1,18 +1,27 @@
 <mj-section padding="0px">
   <mj-column>
     <mj-text>Hi {{ user.name|v2esc }},</mj-text>
-    <mj-text><b>{{ from_user.name|v2esc }} just wrote a reference for you</b> from when you {% if surfed %}surfed with{% else %}hosted{% endif %} them.</mj-text>
+    <mj-text
+      ><b>{{ from_user.name|v2esc }} just wrote a reference for you</b> from
+      when you {% if surfed %}surfed with{% else %}hosted{% endif %}
+      them.</mj-text
+    >
```

Comment:
I'm not sure what you did here?

Reply:
I had some formatOnSave stuff messing this up but I just changed the settings to make sure it only formats on the javascript, typescript, etc files.

Reply:
It's resolved now.